packages/seacas/libraries/ioss/src/Ioss_Region.C

```diff
@@ -3055,4 +3105,28 @@ namespace Ioss {
       topologyObserver->reset_topology_modification();
     }
   }
+
+  std::string Region::get_group_name() const
```

Comment:
Is this needed by external clients...   Not sure it makes sense to expose the concept of a "group" to external clients since it is an artifact of how we are using exodus.   Is it possible to isolate groups at the exodus level.  If not, is there some more descriptive name that we can use at the higher levels...

Reply:
It probably doesn't need to exposed externally .. I'll make it a private/function.